vnext/Shared/JSI/ChakraRuntime.h

```diff
@@ -239,13 +239,6 @@ class ChakraRuntime : public facebook::jsi::Runtime, public ChakraApi, ChakraApi
     // ~ChakraPointerValue() should only be invoked by invalidate().
     // Hence we make it private.
     ~ChakraPointerValue() noexcept override {
```

Comment:
>ChakraPointerValue [](http://example.com/codeflow?start=5&length=18)

Why this change is needed? By avoiding the calling JsRelease we effectively causing a memory leak.